posthog/clickhouse/client/execute.py

```diff
@@ -40,13 +40,7 @@
 
 @lru_cache(maxsize=1)
 def default_settings() -> Dict:
-    # On CH 22.3 we need to disable optimize_move_to_prewhere due to a bug. This is verified fixed on 22.8 (LTS),
-    # so we only disable on versions below that.
-    # This is calculated once per deploy
-    if clickhouse_at_least_228():
-        return {"join_algorithm": "direct,parallel_hash", "distributed_replica_max_ignored_errors": 1000}
-    else:
-        return {"optimize_move_to_prewhere": 0}
+    return {"join_algorithm": "direct,parallel_hash", "distributed_replica_max_ignored_errors": 1000}
```

Comment:
@fuziontech just making sure this was intentional since it doesn't seem to be related

Reply:
it is - we are way beyond 22.8 at this point so this was no longer needed at all

Reply:
Honestly once I have a bit of time I need to go through all of the CH client settings and clean up everything. There's a lot of built up crud in here from different CH versions